.github/workflows/test.yml

```diff
@@ -1,5 +1,5 @@
 name: test
-on: { pull_request: {} }
+on: { pull_request: {}, push: {} }
```

Comment:
Use `push: { branches: [master, main] }` so you don't end up running CI for every single commit someone pushes to an in-progress PR.